src/__tests__/lib/command/util/organizations-util.test.ts

```diff
@@ -2,8 +2,17 @@ import { jest } from '@jest/globals'
 
 import type {
 	OrganizationResponse,
+	OrganizationsEndpoint,
+	SmartThingsClient,
 } from '@smartthings/core-sdk'
 
+import {
+	ValueTableFieldDefinition,
+} from '../../../../lib/table-generator.js'
+import {
+	createChooseFn,
+	type ChooseFunction,
+} from '../../../../lib/command/util/util-util.js'
 import {
 	InputDefinition,
```

Comment:
As long as this file's being updated, does `InputDefinition` want to be identified as a `type` as well?

Reply:
Yes, yes it does! 😄

Also, so do `selectDef`, `staticDef`,`undefinedDef`, and `createChooseFn`.

Good catch and fixed.